sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
I don’t think to have that only. I mean to put it in the “Let us categorize for you” section a hidden div that you can click a dropdown to see.


-- 
Alex Gerstein
gerstein.alex@gmail.com

On May 24, 2015 at 10:53:46 PM, Scott Gladstone (notifications@github.com) wrote:

Literally a try it yourself section in the bottom of the splash page is what I'd initially suggested. I agree it's not the goal for every user, but the users that are willing to read what the site is about woll also likely want to play with things like the categorizer. I understand that it's not a core service and theoretically only the judges need it, but it's so odd that there's no mention of "use our api to categorize" or "categorize on the fly" anywhere. I also don't think we're prepared to change our splash page to just that because it leaves different assumptions of what's actually inside the app, which is not exactly sorted by or predominantly constructed around categories, as that would indicate.